nsiqcppstyle_util.py

```diff
@@ -59,3 +61,13 @@ def GetSystemKey():
 
 def CmpObjects(a, b):
     return (a > b) - (a < b)
+
+
+def RemoveOuterQuotes(raw_string):
+    final_string = raw_string.strip()
+    if final_string.startswith(SINGLE_QUOTE) and final_string.endswith(SINGLE_QUOTE):
+        final_string = final_string[1:-1].strip()
+    elif final_string.startswith(DOUBLE_QUOTE) and final_string.endswith(DOUBLE_QUOTE):
+        final_string = final_string[1:-1].strip()
+
+    return final_string

```

Comment:
if final_string[0] == final_string[-1] and final_string[0] in [SINGLE_QUOTE, DOUBLE_QUOTE]:
    final_string = final_string[1:-1].strip()
return final_string

Reply:
Done.